skietbaan_fe/src/components/NavbarMenu.js

```diff
@@ -98,6 +98,39 @@ class NavbarMenu extends Component {
 		}
 	}
 
+	isProfilev2() {
+		if (window.location.pathname.endsWith("/profile")) {
+			return <img src={require('../components/navbar-icons/profile-red.png')}
```

Comment:
extract this to a variable by importing it in the beginning 
eg `import profileRed from '../components/navbar-icons/profile-red.png'`
then `src={profileRed}`

Reply:
Do it for all please

Reply:
made changes for all